components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -59,6 +59,22 @@
     validate_worker_config,
 )
 
+# Constants
+COMPONENT_NAMES = (
+    CONTROLLER_TARGET_NAME,
```

Comment:
Just a nit, a target isn't a component. One option is to change this list to COMPONENT_AND_TARGET_NAMES, or we can have two lists, one for individual components and another one for target(s)

Reply:
Is it fair to say a component can always be a target? If so, maybe we can rename is as `TARGET_NAMES` and the usages below would be like we check the user specified "target" against this targets list.